.chloggen/nslaughter-fix-sshcheck-test.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
The changelog can be removed, for changes like this we usually add a skipping changelog label